evm/src/executor/inspector/cheatcodes/fork.rs

```diff
@@ -246,3 +255,107 @@ fn create_fork_request<DB: DatabaseExt>(
     };
     Ok(fork)
 }
+
+fn eth_getlogs<DB: DatabaseExt>(data: &mut EVMData<DB>, inner: &EthGetLogsCall) -> Result {
+    let url = data.db.active_fork_url().ok_or(fmt_err!("No active fork url found"))?;
+    if inner.0 > U256::from(u64::MAX) || inner.1 > U256::from(u64::MAX) {
+        return Err(fmt_err!("Blocks in block range must be less than 2^64 - 1"))
+    }
+    if inner.3.len() > 4 {
+        return Err(fmt_err!("Topics array must be less than 4 elements"))
+    }
```

Comment:
I think we could put `4` in a constant along with a comment as to why it needs to be les than 4, to be extra clear